apps/next-evans/pages/friend/[friend_name].tsx

```diff
@@ -0,0 +1,68 @@
+import { PrismaClient } from '@prisma/client';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import FeaturedQuoteBlock from '@/components/pages/friend/FeaturedQuoteBlock';
+import FriendBlock from '@/components/pages/friend/FriendBlock';
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const prisma = new PrismaClient();
+  const allFriends = await prisma.friends.findMany();
+  const paths = allFriends.map((friend) => ({
+    params: {
+      friend_name: friend.slug,
+    },
+  }));
+
+  return {
+    paths,
+    fallback: false,
+  };
+};
+
+export const getStaticProps: GetStaticProps<Props> = async (context) => {
+  const prisma = new PrismaClient();
+  if (typeof context.params?.friend_name !== `string`) {
+    return {
+      notFound: true,
+    };
+  }
+  const friend = await prisma.friends.findFirst({
+    where: { slug: context.params.friend_name },
```

Comment:
you'll need to start thinking in terms of 2 builds -- one per language. so, the correct query here would be for the friend slug, but _also_ specifying the language that we're building in. there is a `lang` column on the `friends` table, for this purpose.  use the env variable i prototyped in the index route.